demisto_sdk/__main__.py

```diff
@@ -225,7 +225,8 @@ def main(ctx, config, version, release_notes, **kwargs):
             if last_release and __version__ != last_release:
                 logger.info(
                     f"[yellow]however version {last_release} is available.\n"
-                    f"To update, run pip3 install --upgrade demisto-sdk[/yellow]"
+                    f"To update, run pip3 install --upgrade demisto-sdk "
+                    f"or poetry install (in case you work in a virtual environment)[/yellow]"
```

Comment:
1. doesn't pip work in `venv`s? 🤔 
2. I think `poetry install` should be when working on `demisto/content`, or any repo that has `pyproject.toml`. Let's find a simple, correct phrasing for this case.

Reply:
1. It works (poetry install will update all dependencies).
2. Correct.

So I'm closing this PR (since this is not required).